webapp/components/notifications/NotificationList/NotificationList.vue

```diff
@@ -24,8 +24,8 @@ export default {
     },
   },
   methods: {
-    markAsRead(notificationId) {
-      this.$emit('markAsRead', notificationId)
+    markAsRead(notificationSourceId) {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:43:02Z" title="Friday, August 30th 2019, 2:43:02 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -24,8 +24,8 @@ export default {
     },
   },
   methods: {
-    markAsRead(notificationId) {
-      this.$emit('markAsRead', notificationId)
+    async markAsRead(notificationSourceId) {
```


why have you adding async if there is no accompanying await?
shouldn't the `async/await` in the parent component `NotificationMenu` be sufficient to wait till the `markAsRead` is finished?